modules/doc/content/getting_started/installation/inl_hpc_install_moose.md

```diff
@@ -0,0 +1,99 @@
+# INL HPC Cluster
+
+This page aims at simplifying the installation process on INL computing clusters.
+The [general cluster instructions](hpc_install_moose.md) may also be used, but these
+instructions will save some steps.
+
+!alert note
+Falcon is planned to be decommissioned this January 2022.
+
+!alert warning
+Do not mix installation workflows (unless this page tells you to).
+All you will get is headaches and compilation errors.
+
+## Module-based installation
+
+### Step 1: load some compilers
+
+In order to compile MOOSE and its applications, we first need to load compilers & compiling tools.
+The following modules are recommended for each of INL's clusters:
+
+- Falcon: `module load use.moose MVAPICH2 CMake`
+
+- Lemhi: `module load use.moose mvapich2 cmake` (if you want to use a PETSc module, load PETSc now to get the right mvapich2)
+
+- Sawtooth: `module load use.moose mvapich2 cmake` (same advice as above for PETSc)
+
+
+!alert note
+If you use HPC modules, you will need to load them every time you log in! One solution is to
+modify the `.bashrc` file and add these `module load` commands there.
+
+### Step 1a: make sure those compilers are used
+
+Depending on how the MPI modules were built (and 99% of the time, this is not needed),
+it may be that the compilers loaded will not be automatically used when compiling.
+To make sure of this, we export those environment variables.
+
+```
+  export CC=mpicc CXX=mpicxx
+```
```

Comment:
FC=mpif90

Reply:
We really should auto this instead of asking set environment. 

I will do that soo